models/idea/index.js

```diff
@@ -274,6 +274,40 @@ class Idea extends Model {
     const cursor = await this.db.query(query, params);
     return await cursor.all();
   }
+
+
+  /**
+   * Read ideas commented by specified users
+   * @param {string[]} minimum rate - minimal query rate
+   * @param {integer} offset - pagination offset
+   * @param {integer} limit - pagination limit
+   * @returns {Promise<Idea[]>} - list of found ideas
+   */
+  static async findHighlyRated(rateBottomLimit, { offset, limit }) {
+    const query = `
+      FOR idea IN ideas
+        FOR vote IN votes
+        FILTER idea._id == vote._to
+        // group by idea id
+        COLLECT id = idea
```

Comment:
Calling idea `id` may be confusing. How about `i` or `collectedIdea` or renaming `idea` to `i` above and using `idea` here? Just `id` is confusing with actual id of something.

Reply:
Ok, to be changed.